oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+  the project in the last 6 months.  The types of contributions that
+  are relevant will match those defined in the `core contributor
+  program`_
+* Abide by the `Open edX Code of Conduct
+  <https://openedx.org/code-of-conduct/>`_
+* Be a core contributor for the repository in question who has
+  demonstrated excellence in the role of core contributor.  From
+  time-to-time, in the case of a strong candidate and project need,
+  core contributor status and maintainer status may be conferred at
+  the same time.
+
+Bootstrapping
+-------------
+In order to ensure an orderly roll-out of the formal maintainer role,
+existing 2U staff in the owner role will transition into the role of
+maintainer for the components that they own.
+
+Community members who are acting in informal maintainer roles, will
+also be asked to become named maintainers.
+
+A list of all components without maintainers will be created and
+published.
+
+An asynchronous maintainer scrum-of-scrums will be created as a Slack
+channel.
+
+Standards for measuring the quality of repositories will be rolled out
+across all repositories.  These standards can benefit from work done
+at edX to measure `repository health <https://github.com/openedx/edx-repo-health>`_.
+
+Becoming a Maintainer
+---------------------
+After the bootstrapping phase, the Open edX project will need
+processes for adding additional maintainers, replacing maintainers,
+and ensuring that new components are maintained.
+
+**This section should be congruent with the Core Contributor process**
+
+A new maintainer can be added in any of the following ways:
+
+* An existing maintainer can request that a new person become a
+  component maintainer either in conjunction with the existing
+  maintainer or replacing them.  If there is no objection from
+  existing maintainers, the role will be conferred.
+* In the case of vacancies, the nominations will be made to the
+  scrum-of-scrums to fill the vacancy.  If there is not objection from
+  existing maintainers, the role will be conferred.
+* If there consensus cannot be reached,
+* In the case of new repositories, there should be a designated
+  maintainer named before the repository is created.
+
+Removing a Maintainer
+---------------------
+From time-to-time it will be necessary to remove a component
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project that the can no longer continue in the role
+* A maintainer has been unreachable for more than 30 days
+* A maintainer has violated the project `Code of Conduct`_
+* A maintainer hasn't met project quality standards consistently despite feedback
+  
```

Comment:
I've added some proposed language about removal.

Reply:
Would the component/repo then have an "unmaintained" status? Or would the maintainer become some default person/group until another maintainer is found?

Reply:
I would say that it would enter an "unmaintained" status.  This confronts that fact that there is not group with unlimited spare capacity to take on orphans.  It would catalyze us to find a new maintainer rather than make do.